src/engine/strat_engine/metadata.rs

```diff
@@ -227,17 +234,22 @@ impl StaticHeader {
 pub struct MDARegions {
     // Spec defines 4 regions, but regions 2 & 3 are duplicates of 0 and 1 respectively
     pub region_size: Sectors,
-    pub mdas: [MDAHeader; NUM_PRIMARY_MDA_REGIONS as usize],
+    pub mdas: [MDAHeader; NUM_PRIMARY_MDA_REGIONS],
 }
 
 impl MDARegions {
+    /// Calculate the offset from start of device for an MDARegion.
+    fn mda_offset(index: usize, per_region_size: Bytes) -> u64 {
```

Comment:
The result of the function is used exclusively as an argument to ```SeekFrom()``` which requires a ```u64```, representing bytes, as an argument.

It might make more sense to return it as Bytes and then dereference the value where used in ```SeekFrom()``` calls, but, since this is a private function, I didn't go to the trouble.